### ISSUE TITLE
ort-web Error: invalid input shape when there is a torch.nn.BatchNorm1d layer in the network

### ISSUE THREAD

**Issue by ffletcherr** [microsoft/onnxruntime#10437]
**Describe the bug**
When I convert a network from pytorch to onnx that has an `torch.nn.BatchNorm1d` and use it in a JavaScript Web app with `webgl` backend this error will be raised (`wasm` backend is okay):
```js
ort-web.min.js:6 Uncaught (in promise) Error: invalid input shape.
    at u (ort-web.min.js:6:227241)
    at Object.t.batchNormalization [as impl] (ort-web.min.js:6:225921)
    at eval (ort-web.min.js:6:357179)
    at t.Profiler.event (ort-web.min.js:6:372373)
    at eval (ort-web.min.js:6:357139)
```

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 18.04):
- ONNX Runtime installed from (source or binary): npm
- ONNX Runtime version: onnxruntimeweb 1.10.0
- Python version: 3.8.12
- Visual Studio version (if applicable): -
- GCC/Compiler version (if compiling from source): -
- CUDA/cuDNN version: -
- GPU model and memory: NVIDIA GeForce GTX 1650 Ti

**To Reproduce**
You can use this code to create .onnx model.
```python
import torch
from torch import nn

class myNet(nn.Module):
    def __init__(self):
        super(myNet, self).__init__()
        self.layer1 = nn.Linear(256,128)
        self.bn1 = nn.BatchNorm1d(128)
        self.layer2 = nn.Linear(128,64)
        self.bn2 = nn.BatchNorm1d(64)
        self.layer3 = nn.Linear(64,2)
    def forward(self,x):
        x = self.layer1(x)
        x = self.bn1(x)
        x = self.layer2(x)
        x = self.bn2(x)
        x = self.layer3(x)
        return x

if __name__ == '__main__':
    net = myNet()
    net.eval()
    device = "cuda" if torch.cuda.is_available else "cpu"
    net.to(device)
    dummy_input = torch.ones(size=(1,256)).to(device)
    dummy_output = net(dummy_input)
    onnx_path = "simple.onnx"
    torch.onnx.export(net, dummy_input, onnx_path, verbose=True,
                    input_names=["input"], output_names=["output"])
```
and use this script to load and test .onnx model in js:
```js
net = await ort.InferenceSession.create("simple.onnx",{executionProviders: ["webgl"]}); 
const tensor = new ultra.ort.Tensor('float32', new Float32Array(256), [1,256]);
const output = await net.run({["input"]:tensor});
```
Also you need an html file which can use something like this : [quick-start_onnxruntime-web-script-tag](https://github.com/microsoft/onnxruntime-inference-examples/blob/main/js/quick-start_onnxruntime-web-script-tag/index.html)

**Comment by mauricewells** [comment]
i reached at issue like you

**Comment by mauricewells** [comment]
who can explain such issue?

**Comment by naorsabag** [comment]
Has the same issue, although for me it doesn't work on wasm either, but it might be unrelated.
Using onnxruntime through cdn @1.11.0/dist/ort.min.js, and onnx.py latest. opset_version=16
It works fine when I remove the batchnorm.

**Error message** - "Uncaught (in promise) Error: invalid input shape." (from [ batch-normalization.ts:85:11](https://github.com/microsoft/onnxruntime/blob/0c6037b5abe571fc43a55ef7a9d2f846820fbe5d/js/web/lib/onnxjs/backends/webgl/ops/batch-normalization.ts#L85),  fails on condition X.dims.length < 3 )
